src/unitxt/operators.py

```diff
@@ -777,7 +777,7 @@ class Apply(InstanceOperator):
     Args:
         function (str): name of function.
         to_field (str): the field to store the result
-        additional arguments are field names passed to the function
+        additional arguments are field names passed in
```

Comment:
```suggestion
        any additional arguments are field names whose values will be passed directly to the specified function
```

Reply:
Hi- thanks for that -- I think it makes sense but I think the code has an issue with using the word "function" at the end -- which is why at first the word function got out of the parentheses -- we can switch the word order to share the same meaning and not end the sentence with the word in. Let me see... I'll propose another change incorporating your words. I can understand if it didn't seem clear enough without them. Thanks!

Reply:
p.s.: you can see the issue when you look at the rendered web page -- as opposed to looking at it in the python file